frontend/packages/console-shared/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -1,8 +1,37 @@
 import * as _ from 'lodash';
 import * as React from 'react';
 import * as fuzzy from 'fuzzysearch';
-import { Dropdown, FirehoseResult, LoadingInline } from '@console/internal/components/utils';
-import { K8sResourceKind } from '@console/internal/module/k8s';
+import {
+  Dropdown,
+  FirehoseResult,
+  LoadingInline,
+  ResourceIcon,
+} from '@console/internal/components/utils';
+import {
+  K8sResourceKind,
+  referenceForModel,
+  K8sKind,
+  modelFor,
+  referenceFor,
+} from '@console/internal/module/k8s';
+
+type DropdownItemProps = {
+  model: K8sKind;
+  name: string;
+};
+
+const DropdownItem: React.FC<DropdownItemProps> = ({ model, name }) => (
+  <span className="co-resource-item">
+    <span className="co-resource-icon--fixed-width">
+      <ResourceIcon kind={referenceForModel(model)} />
+    </span>
+    <span className="co-resource-item__resource-name">
+      <span className="co-resource-item__resource-api co-truncate show co-nowrap small">
+        {name}
+      </span>
+    </span>
+  </span>
+);
```

Comment:
All these classes are part of `resource-dropdown`.
We should not be reusing classes like this because that component could change and will break this component.
If we keep this here we need a tech debt item, and a comment, to cover fixing this issue.

Reply:
Have updated the classes, just used classes on PF i.e `co-resource-item` and common styles